wolfSSL/src/chapter02.md

```diff
@@ -536,10 +716,180 @@ Enables Online Certificate Status Protocol (OCSP) support.
 
 Builds even more OpenSSL compatibility into the library, and enables the wolfSSL OpenSSL compatibility layer to ease porting wolfSSL into existing applications which had been designed to work with OpenSSL. It is off by default.
 
+#### HAVE_EXT_CACHE
+
+OpenSSL compat layer. Need OPENSSL_EXTRA to be defined as well.
```

Comment:
Is this really tied to the openssl compatibility layer? This enables a feature support use of an external session cache (vs internal one).